dbt_project.yml

```diff
@@ -1,5 +1,5 @@
 name: 'jira'
-version: '0.3.0'
+version: '0.3.1'
```

Comment:
Would this not be a breaking change? If an existing user installs this release then their incremental runs will fail. We should upgrade this to `v0.4.0` to account for the breaking change.